syntax = "proto3";

package example;

service Greeter {
  rpc Units (UnitRequest) returns (UnitReply) {}
  rpc Units2 (UnitText) returns (UnitText) {}
}

message UnitRequest {
  string name = 1;
}

message UnitReply {
  string message = 1;
}

message UnitText{
  string text = 1;
}